common/ethereum/src/eth_block.rs

```diff
@@ -44,6 +45,10 @@ pub struct EthBlock {
     pub transactions_root: EthHash,
     pub uncles: Vec<EthHash>,
     pub base_fee_per_gas: Option<U256>,
+    pub withdrawals_root: Option<EthHash>,
+    pub blob_gas_used: Option<U256>,
+    pub excess_blob_gas: Option<U256>,
+    pub parent_beacon_block_root: Option<EthHash>,
```

Comment:
It's worth adding a note that these four new fields are `EIP4844` specific and hence they're optional, for future reference.